benchmarks/create-pages/gatsby-node.js

```diff
@@ -1,10 +1,11 @@
 const NUM_PAGES = parseInt(process.env.NUM_PAGES || 5000, 10)
 
+const blankTemplate = require.resolve(`./src/templates/blank.js`)
```

Comment:
This (not counted in 10% dropped) also took a second or two off the total time heh. Apparently there's a lot of things that start to take up human-noticable time if you call them 100k times.

Reply:
Then we should double check that our docs use that pattern :)

Reply:
👍